assignments/prototypes.js

```diff
@@ -102,9 +217,12 @@
   console.log(archer.greet()); // Lilith offers a greeting in Elvish.
   console.log(mage.takeDamage()); // Bruce took damage.
   console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+  console.log(hero.bigOldSlash(villain));
+  console.log(villain.leech())
```

Comment:
```suggestion
  console.log(villain.leech(hero))
```
Just need to call this with an argument